src/GovUk.Education.ExploreEducationStatistics.Content.Model/UserPublicationInvite.cs

```diff
@@ -0,0 +1,35 @@
+#nullable enable
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+{
+    public class UserPublicationInvite
+    {
+        [Key] 
+        [Required]
+        public Guid Id { get; set; }
+
+        [Required] public string Email { get; set; } = null!;
+
+        [Required] public Publication Publication { get; set; } = null!;
+
+        [Required]
+        public Guid PublicationId { get; set; }
+        
+        [Required]
+        public PublicationRole Role { get; set; }
+        
+        public bool Accepted { get; set; }
```

Comment:
> Do we need this? I know we set it for UserReleaseInvites but UserInvite has its own Accepted field which should be all we need. Tbh, even that field is a little redundant because you could just check if a user exists on the system or not to determine if the invite was accepted.

Investigating a little, it seems like the original intention of `Accepted` on `UserReleaseInvites` was that an existing user could be invited to have a new release role - an email is also sent when this happens. Not sure if `UserReleaseInvites` are still used in this case - if the user already exists on the system, then a new `UserReleaseRole` can be created immediately - I believe that's what happens with the `ContributorInvite` stuff. Looking at the code, I don't see where `UserReleaseInvites` get processed if the user already exists. And `UserRoleService#AddReleaseRole` instantly creates the role - it doesn't create a UserReleaseInvite.

So I'm going to go ahead and delete `Accepted` from `UserReleaseInvites` and `UserPublicationInvites`, and anywhere `Accepted` was set, I'll instead remove the release/publication invite entirely.

Reply:
`ReleaseInviteService#CreateExistingUserContributorInvite` was creating accepted `UserReleaseInvites` after creating the `UserReleaseRoles` themselves - so it looks like it was being used for auditing. Anyway, I'm going to continue and remove the `Accepted` columns.